FreeRTOS-Plus/Demo/FreeRTOS_Plus_TCP_Echo_Posix/TCPEchoClient_SingleTasks.c

```diff
@@ -195,6 +195,7 @@
 
                     /* Add in some unique text at the front of the string. */
                     sprintf( pcTransmittedString, "TxRx message number %u", ulTxCount );
+                    pcTransmittedString[strlen(pcTransmittedString)] = "-";
```

Comment:
sprintf method will insert '\0' NULL terminator into the string, which causes the demo to send and receive "TxRx message number %u" only, instead of "TxRx message number %u" plus the pcTransmittedString. This line of code is to replace  '\0' with '-' so that the full pcTransmittedString can be sent and used for string comparison later.